src/util/d.ts

```diff
@@ -16,28 +16,33 @@ export type DOptions<S extends WidgetState, O extends WidgetOptions<S>> = VNodeP
 
 type Children = (DNode | VNode | null)[];
 
-function d(tagName: string, options?: VNodeProperties, children?: (DNode | VNode | null)[]): HNode;
+function d(tagName: string, optionsOrChildren?: VNodeProperties | Children, children?: Children): HNode;
 function d<S extends WidgetState, W extends Widget<S>, O extends WidgetOptions<S>>(factory: ComposeFactory<W, O>, options: O): WNode;
 function d<S extends WidgetState, W extends Widget<S>, O extends WidgetOptions<S>>(
 	tagNameOrFactory: TagNameOrFactory<S, W, O>,
-	options: DOptions<S, O> = {},
+	optionsOrChildren: DOptions<S, O> = {},
 	children: Children = []): DNode {
```

Comment:
I should have caught this before, but the closing paran should be outdented so it is clear where the arguments end:

```ts
	children: Children = []
): DNode {
```